src/js/oTable.js

```diff
@@ -25,9 +25,16 @@ function OTable(rootEl) {
 		const tableRows = Array.from(this.rootEl.getElementsByTagName('tr'));
 
 		this.tableHeaders.forEach((th, columnIndex) => {
+			th.setAttribute('tabindex', "0");
+
 			const listener = this._sortByColumn(columnIndex);
 			this.listeners.push(listener);
 			th.addEventListener('click', listener);
+			th.addEventListener('keydown', (event) => {
+				if (event.code === "Space" || event.code === "Enter") {
```

Comment:
Keyboard event code property doesn't exist in IE, Edge or Android and we don't have a polyfill for it neither :-(

Reply:
🙈 Thanks Jake - I should have known this tbh.